recipes/clickhouse-sqlalchemy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: clickhouse-sqlalchemy
+  version: "0.0.3"
+
+source:
+  fn: clickhouse-sqlalchemy-0.0.3.tar.gz
+  url: https://pypi.python.org/packages/cb/3c/3720527c06c31ca1c2c1f0e33af30b3dc099e6a69d73357063a5f19126df/clickhouse-sqlalchemy-0.0.3.tar.gz
+  md5: b3e9eef9cd4960d1d7de47c843e7e987
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
I believe that only `setuptools` and `python` are `build` requirements of this package. The `--single-version-externally-managed` flag means that the `install_requires` aren't necessary unless `setup.py` causes them to be imported.

Reply:
Should I remove `--single-version-externally-managed` from each of the packages?

Reply:
No, you want that flag if `setup.py` uses `setuptools`. What I was trying to say is that the flag means that you don't need to list those other packages as `build` dependencies.  They should still appear as `run` dependencies.